cloudslang-content-verifier/src/main/java/io/cloudslang/lang/tools/build/tester/TriggerTestCaseEventListener.java

```diff
@@ -121,6 +128,10 @@ public ReturnValues getExecutionReturnValues(){
     private void printFinishEvent(Map<String, Serializable> data) {
```

Comment:
I think that this method should be removed from here. You can print the log in the testRunner. The event listener is not the right location for it. 
This way you can also remove the expectedResult variable